README.md

```diff
@@ -11,6 +11,14 @@ It has been tested on the following rplidars:
 
 To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/modular-resources/configure/#add-a-module-from-the-viam-registry) and select the `viam:lidar:rplidar` model from the [`rplidar` module](https://app.viam.com/module/viam/rplidar).
 
+### Requirements 
+
+The `rplidar` module is distributed as an AppImage.
+AppImages require FUSE version 2 to run.
+See [FUSE troubleshooting](/appendix/troubleshooting/#appimages-require-fuse-to-run) for instructions on installing FUSE 2 on your system if it is not already installed.
```

Comment:
The link leads to a "404 - page not found" error: https://github.com/viamrobotics/rplidar/blob/sguequierre-patch-1/appendix/troubleshooting/#appimages-require-fuse-to-run

Should it be https://docs.viam.com/appendix/troubleshooting/#appimages-require-fuse-to-run instead?

```suggestion
See [FUSE troubleshooting](https://docs.viam.com/appendix/troubleshooting/#appimages-require-fuse-to-run) for instructions on installing FUSE 2 on your system if it is not already installed.
```

Reply:
Oh yes that is right! Thank you!!